src/messaging/messaging-errors.ts

```diff
@@ -67,21 +67,22 @@ export function createFirebaseError(err: HttpError): FirebaseMessagingError {
  */
 export function getErrorCode(response: any): string | null {
   if (validator.isNonNullObject(response) && 'error' in response) {
-    if (validator.isString(response.error)) {
-      return response.error;
+    const error = (response as any).error;
```

Comment:
Is this because `isNoneNullObject()` asserts the type of `response` to be something other than `any`?